crates/sql-sink/src/backoff.rs

```diff
@@ -0,0 +1,22 @@
+use std::time::Duration;
+
+pub(crate) enum Backoff {
+    Fib(u64, u64),
```

Comment:
We already have such thing in the common library https://docs.rs/fluvio-future/latest/fluvio_future/retry/struct.FibonacciBackoff.html